CHANGELOG.md

```diff
@@ -2,10 +2,36 @@
 
 Mapbox welcomes participation and contributions from everyone.
 
-# 10.0.0-rc.1 June 10, 2021
+# 10.0.0-rc.2 June 23, 2021
 
 **The Mapbox Maps SDK for Android has moved to release candidate status and is now ready for production use.**
 
+## Features ✨ and improvements 🏁
+* Add Persistent Layer API so that the tagged layer and its associated resources would remain when a style is reloaded. This improves performance of Annotation and Location Component Plugin during the style change. ([#368](https://github.com/mapbox/mapbox-maps-android/pull/368))
+* Introduce experimental `Style#addPersistentLayer`, `Layer#isPersistent`, `Style#addPersistentStyleLayer`, `Style#addPersistentStyleCustomLayer` and `Style#isStyleLayerPersistent` APIs. ([#422](https://github.com/mapbox/mapbox-maps-android/pull/422))
```

Comment:
```suggestion
* Introduce experimental `Style#addPersistentLayer`, `Layer#isPersistent`, `Style#addPersistentStyleLayer`, `Style#addPersistentStyleCustomLayer` and `Style#isStyleLayerPersistent` APIs. ([#422](https://github.com/mapbox/mapbox-maps-android/pull/422), [#368](https://github.com/mapbox/mapbox-maps-android/pull/368))
```

Reply:
@pengdev perhaps we could squash this point with the first one?

Reply:
yes, I think so.

Reply:
It's the same feature, but landed in two PRs.